e2e/clone_test.go

```diff
@@ -131,17 +120,7 @@ func testPVCClone() {
 
 		By("confirming that the file exists in the cloned volume")
 		Eventually(func() error {
-			_, _, err := kubectl("get", "pvc", thinClonePVCName, "-n", nsCloneTest)
-			if err != nil {
-				return fmt.Errorf("failed to get PVC. err: %v", err)
-			}
-
-			_, _, err = kubectl("get", "pods", thinClonePodName, "-n", nsCloneTest)
-			if err != nil {
-				return fmt.Errorf("failed to get Pod. err: %v", err)
-			}
-
-			stdout, _, err := kubectl("exec", "-n", nsCloneTest, thinClonePodName, "--", "cat", writePath)
+			stdout, err := kubectl("exec", "-n", nsCloneTest, thinClonePodName, "--", "cat", writePath)
```

Comment:
If the pod and the pvc are not ready, exec will be failed, so the checks are not needed.